README.md

```diff
@@ -118,7 +145,24 @@ These scripts will build the add-on to work with dev, stage, or prod servers.
  * `npm run build:prod`: https://relay.firefox.com/
 
 ### Distributing
+
+#### Add-on Submission Process
+
+See internal documentation [here](https://mozilla-hub.atlassian.net/wiki/spaces/SECPRV/pages/27034372/Firefox+Relay+Add-on+Submission+Process). This document outlines how to submit and release the Firefox Relay add-on (extension) to the different add-on stores.
+
+Process summary: 
+
+![](https://i.imgur.com/rimgrmE.png)
+
+1. Cut Release: Create a new release on GitHub (new tag) with new version number.
+2. Open ADDONOPS ticket: Create a new Jira ticket for the ADDONOPS team using their form, filling out all the particulars. 
+3. QA Greenlight: Link to QA comment or have QA comment directly on the ticket with greenlight. 
+4. Approval: If approval is given, you will be asked to submit to AMO.
+5. Submit to AMO: Submit the add-on to AMO. <br> 
+   **WARNING:** make sure to choose the right options.
```

Comment:
Q: Are there specific options that are problematic or worthy of double checking?
This might not be an entirely... helpful comment.